cms/tests/test_extensions.py

```diff
@@ -31,8 +31,9 @@ def test_register_extension(self):
         page_extension = self.get_page_extension_class()
 
         # register first time
-        extension_pool.register(page_extension)
+        register_extension_result = extension_pool.register(page_extension)
```

Comment:
The more we test the code in the way it's used in practice, the better.
I guess 99% of the decorator usage is with ``@``-syntax, and in the test you can just check that the extension has been properly registered.